generated_interfaces.go

```diff
@@ -212,6 +213,7 @@ type Page interface {
 	ExpectedDialog(cb func() error) (Dialog, error)
 	Fill(selector, text string, options ...FrameFillOptions) error
 	Focus(expression string, options ...FrameFocusOptions) error
+	Frame(name string, url interface{}) Frame
```

Comment:
Maybe an option struct would be better here, but unsure, open for your thoughts on that. Since it would look weird when the user only uses the URL and he needs to pass an empty string to the name parameter.

Reply:
Agree. Changes made. Please review, thanks.